server/controllers/categories.js

```diff
@@ -13,6 +13,10 @@ export async function create(req, res) {
         return sendErr(res, `when creating a category: ${cat}`, 400,
                        'Missing category title');
 
+    if (typeof cat.color === 'undefined')
+        return sendErr(res, `when creating a category: ${cat}`, 400,
+                      'Missing category color');
```

Comment:
nit: please align the opening quote with the `r` of the line above, not with the parenthesis.